cli/src/commands/create-mint/index.ts

```diff
@@ -0,0 +1,71 @@
+import { Args, Command, Flags } from "@oclif/core";
+import {
+  CustomLoader,
+  getPayer,
+  getRpc,
+  getSolanaRpcUrl,
+} from "../../utils/utils";
+import { createMint } from "@lightprotocol/compressed-token";
+import {
+  ConfirmOptions,
+  Connection,
+  Keypair,
+  PublicKey,
+  Signer,
+} from "@solana/web3.js";
+import { confirmTx } from "@lightprotocol/stateless.js";
+class CreateMintCommand extends Command {
+  //light create-mint -mint Optional<Keypair> --mint-decimals <u8>
+  static summary = "Create a new compressed token mint";
+
+  static examples = ["$ light create-mint --mint-decimals 2"];
+
+  static flags = {
+    // ...standardFlags,
+    // ...confirmOptionsFlags,
+    "mint-decimals": Flags.integer({
+      description: "Mint decimals.",
+      required: false,
+      default: 2,
+    }),
+  };
+
+  static args = {};
+
+  async run() {
+    const { args, flags } = await this.parse(CreateMintCommand);
+
+    const loader = new CustomLoader(`Performing create-mint...\n`);
+    loader.start();
+
+    try {
+      let mintDecimals = flags["mintDecimals"];
+      if (!mintDecimals) {
+        mintDecimals = 2;
+      }
+      const payer = getPayer();
+
+      const randomMint = Keypair.generate();
+      const connection = new Connection(getSolanaRpcUrl());
+
+      // TODO: remove this, it's just for testing
+      const sig = await connection.requestAirdrop(payer.publicKey, 2e9);
+      await confirmTx(connection, sig);
+
+      const { mint, transactionSignature } = await createMint(
+        connection,
+        payer,
+        payer.publicKey,
```

Comment:
we should allow users to pass in a separate `mintAuthority` (by default this would usually be the same as `payer`, which in turn should be read from the default destination keypair.json like solana cli does it iirc)